modules/cnf-deploying-the-du-infrastructure-profile.adoc

```diff
@@ -0,0 +1,267 @@
+// CNF-950 4.7 Deploying the DU infrastructure profile
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-deploying-the-du-infrastructure-profile_{context}"]
+= Deploying the DU infrastructure profile
+
+== Creating the Performance Addon Operator and DU performance profile
+
+. Create the `kustomization.yaml` file, for example:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - performance-profile.yaml
+----
+
+. Create the `performance-profile.yaml` file, for example:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v1
+kind: PerformanceProfile
+metadata:
+# This profile is for typical lab DELL R640 with 2 x 26 pcores
+  name: perf-example
+spec:
+  additionalKernelArgs:
+  - nosmt
+  cpu:
+    # Temp. workaround for RT kernel bugs that consume too much
+    # CPU power: add more isolated cores
+    isolated: "1,3,5,7,9-51"  <1>
+    reserved: "0,2,4,6,8"     <2>
+  hugepages:
+    defaultHugepagesSize: 1G
+    pages:
+    - count: 16   <3>
+      size: 1G
+      node: 0
+  nodeSelector:
+    # Pay attention to the node label, create MCP accordingly
+    node-role.kubernetes.io/worker-cnf: ""
+  numa:
+    topologyPolicy: "restricted"
+  realTimeKernel:
+    # For CU should be false
+    enabled: true
+----
+<1> Configure this line based upon the customer CPU hardware selected to run the DU.
+<2> Configure this line based upon the customer CPU hardware selected to run the DU.
+<3> Configure this line based upon the customer memory configuration selected to run the DU.
+
+== Creating the PTP operator and slave profile
```

Comment:
We're avoiding the usage of master/slave per the conscious language initiative. Is there alternate language you can use? Recommended replacements for 'slave' are: 'secondary', 'subordinate', 'replica', 'responder', 'follower', 'proxy', or
+'performer' per https://lkml.org/lkml/2020/7/4/229

Reply:
At this time, the code cannot be changed due to upstream ownership conflicts. The approved Conscious Langauge disclaimer will be posted in the documentation.